contents/tutorials/migrate-eu-cloud.md

```diff
@@ -0,0 +1,52 @@
+---
+title: How to migrate to EU Cloud
+sidebar: Docs
+showTitle: true
+author: ['ian-vanagas']
+date: 2022-10-10
+featuredImage: ../images/tutorials/banners/migrate-eu-cloud.png
+topics: ['configuration']
+---
+
```

Comment:
```suggestion

> **Warning:** Migrating events data to PostHog's EU Cloud is a time consuming process, likely to take several days or weeks, and there is currently no way to migrate insights or dashboards. We strongly recommend that users with a large amount of captured events begin by creating a new EU cloud instance and only migrating data at a later point, if strictly needed. 
```

Proposed amend to warn users of impact. @benjackwhite @camerondeleone @simfish85

Alternative here would be to suggest using the replicator, but that feels complex and could/should be a tutorial in itself.

Reply:
> Proposed amend to warn users of impact. @benjackwhite @camerondeleone @simfish85
> 
> Alternative here would be to suggest using the replicator, but that feels complex and could/should be a tutorial in itself.

Reply:
Slightly updated wording @joethreepwood, but good with me.